src/SumoCoders/FrameworkCoreBundle/Resources/assets/coffee/framework.coffee

```diff
@@ -142,19 +142,27 @@ class Framework extends DefaultObject
     )
 
   _initSortable: ->
-    $( '.sortable' ).sortable()
-    handle: 'button'
-    cancel: ''
+    return if (!$.isFunction($.fn.sortable))
```

Comment:
Niet 100% mee akkoord, want dat wil zeggen dat sortable niet geladen is. Niet logischer om eerst te checken of er elementen met class `sortable` zijn?